modules/tinymce/src/themes/silver/test/ts/browser/editor/header/InlineHeaderTest.ts

```diff
@@ -61,7 +61,7 @@ describe('browser.tinymce.themes.silver.editor.header.InlineHeaderTest', () => {
     });
 
   it('TINY-8977: If the editor fits with a wide margin it should not set a width', () =>
-    pRunToolbarWidthTest(500, Optional.none())
+    pRunToolbarWidthTest(500, Optional.some('400px'))
```

Comment:
Since all calls to this function is now Optional.some maybe it would be better to make the input parameter just a string and wrap it in the `pRunToolbarWidthTest`?